### PR TITLE
RELATED: RAIL-3237 Fix DashboardView on tiger

### PR DESCRIPTION
**Pull request by no23reason**
* prevent call to unsupported endpoint
* implement missing necessary endpoint
* improve error messages

JIRA: RAIL-3237

---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3172/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4819/ : FAILURE in 12m 52s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3174/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4822/ : SUCCESS in 12m 38s

File: libs/sdk-backend-tiger/src/utils/api.ts [hunk 1]
```diff
@@ -46,3 +46,19 @@ export const objRefToIdentifier = async (
 
     return matches[1];
 };
+
+/**
+ * Converts ObjRef instances to identifiers. For IdentifierRef returns the identifier as is,
+ * for UriRef calls the backend and gets the identifiers.
+ * @param refs - refs to convert
```
Comment (lupko) [inline libs/sdk-backend-tiger/src/utils/api.ts:53]: this is not correct right?

Comment (no23reason) [reply libs/sdk-backend-tiger/src/utils/api.ts]: Reworded

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/3178/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4826/ : SUCCESS in 12m 39s

**Review by lupko — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1238

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/948/ : SUCCESS in 58s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1217/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/4827/ : SUCCESS in 12m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-el7/332/ : SUCCESS in 8m 42s (non-voting)

**Pull request merged**